test/app-luatest/fiber_deadline_test.lua

```diff
@@ -0,0 +1,73 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_each(function()
+    g.server = server:new({alias = 'default'})
+    g.server:start()
```

Comment:
You don't need to start a new server for this test.

Reply:
yep, fixed